test/FESpacesTests.jl

```diff
@@ -7,64 +7,16 @@ using GridapDistributed
 using PartitionedArrays
 using Test
 
-function main(parts)
```

Comment:
Hi @amartinhuertas ! From this diff it is difficult to see the significant changes in this test.  Which are the main changes here? I guess that the previous functionality is still been tested, right?

The changes in src look pretty minor. Thus, OK!

Reply:
> Which are the main changes here?

Basically I encapsulated a piece of test code within a function (`assemble_tests`) so that it can be leveraged both with D and D-1-dim triangulations.

> The changes in src look pretty minor. Thus, OK!

Not ready to be merged. See https://github.com/gridap/GridapDistributed.jl/pull/86#issue-1181584024. There are still issues to be solved and questions to be answered. :-)